lib/runner/matchers/filename.js

```diff
@@ -114,4 +116,4 @@ class FilenameMatcher {
   }
```

Comment:
I had considered this
```javascript
match(fullPath) {
  return this.filterPattern.some(pattern => fullPath.includes(pattern) || minimatch(fullPath, pattern));
}
```
but the PR as it is felt more in keeping with the current style, however happy to change if you prefer

Reply:
Yes, this is significantly more elegant and also better performance.